### ISSUE TITLE
No active keyboard focus is present on the Left pane (Tree View pane) of Inspector window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#290]
### Pre-requisites:
Launch the Visual Studio for Mac and Sign-in with Enterprise account.

Create projects:

1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:
1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Left pane (Tree View pane) of the inspector window.
3. Navigate through these tree items using arrow keys.

### Expected Behavior
Active keyboard should be present on Left pane while keyboard user traverses  these tree/table items using arrow keys.
### Actual Behavior
No active keyboard focus is present on Left pane (Tree View pane) of Inspector window.
Note1: User is unable to diffrentiate the keyboard focus while focus is present on the tree view elements and other than these tree view elements.
Note2: The Keyboard user is able to navigate to these tree elements using arrow keys, so no issue to navigate these elements.
### User Impact
The keyboard user is unable to know whether keyboard focus is present on tree view elements or not.
User is unable to know, whether the left pane is getting keyboard the focus or not.

![kb6_leftpaneui-tree_noactivekeyboardfocus](https://user-images.githubusercontent.com/22956130/39986748-441ed27c-5780-11e8-9e49-bcf36253ac5d.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

### Log File

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.4.7

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
